backend/index.js

```diff
@@ -1,66 +1,19 @@
 const express = require('express');
-const cors = require('cors');
 const helmet = require('helmet');
-const HTTP = require('http-status-codes');
-const bodyParser = require('body-parser');
-const session = require('express-session');
-const RedisStore = require('connect-redis')(session);
+const errorHandler = require('./plugins/errorHandler');
 const di = require('./di');
 const router = require('./routes')(di);
 
-const config = require('./config');
-
 const app = express();
 
 app.set('di', di);
 
 app.use(helmet());
+app.use(errorHandler);
```

Comment:
I think it should be after `router` also you have to wrap errors before `router`